### ISSUE TITLE
Can't open albums

### ISSUE THREAD

**Issue by Xergoth** [Quark987/plugin.video.moments#2]
Hi - great addon, thank you!

I can view albums just fine, but have had no luck trying to open any and view the photos within. This also applies to anything auto-tagged, locations etc.

What am I missing here?

Thanks!

**Comment by Quark987** [comment]
Hi,

Can you give some more details? 
- What version of Kodi are you running? 
- What error (if any) do you get?
- What version of Synology are you running?
- Can you maybe show a screenshot of the described behaviour?

It's strange that the core functionality, i.e. viewing photos, doesn't work on your side.

**Comment by Xergoth** [comment]
Hi,

Kodi version: 18.8.0

Synology/ DSM version: 6.1.7 (can't update, long story)

Error message in Kodi: "Synology Moments error: Check the log for more information"

I've attached the Kodi log to potentially help with finding the culprit. Are there any particular settings that need to be active in Synology? Moments is working fine on a tablet and 2 phones.

Thanks very much for your help!

[Kodi log.txt](https://github.com/Quark987/plugin.video.moments/files/6571647/Kodi.log.txt)

**Comment by Quark987** [comment]
Can you see the content of any album or none at all?
For the log you send me it seems like you have an empty album and the code can't handle that yet (error management is not great to be honest). So, is the album you're trying to open indeed empty?

The code can only communicate with the synology using http, https is not supported. So please make sure the port entered in the settings correspond to the http port. Also make sure http is enabled on your synology. Aside from that, it should just work.

**Comment by Xergoth** [comment]
I can see the thumbnails of every album, but can't see the contents of any. I've checked them all, and deleted the one empty album I did have. So no, the album(s) in question isn't/ aren't empty, unfortunately.

I'm indeed using http still over https, and the port used corresponds to that. It's all rather puzzling - I'm sure we can find a solution, just at a loss myself as to what it could be right now.

Does the add-on communicate with the Synology device over the local network like any other device would? It's strange, as I'm not having any issues with anything else. I use the server for all sorts of things.

Many thanks!

**Comment by Quark987** [comment]
Yes, the add-on is communicating using your LAN. It's particularly strange as you are able to see the albums but not their content. Can you also give me the version of Moments you're using?
If you're up for it, let's dig a little deeper. The first attempt will be to verify if the request from a browser matches the one sent by the add-on. Because, ultimately, all the add-on does is replicate these requests. Below are the steps you'll need to take for this.

1. Fire up your favourite browser (Chrome is preferred, as this is what I was using).
2. Go to the web page for Moments
3. If on a Mac, click on View -> Developer -> Developer Tools, on Windows, click on the three dots and choose More tools -> Developer tools
4. In Moments, click on albums and go inside an album. You will then see the content, which you couldn't see using the add-on.
5. Check the requests made by your browser. It will first request a list of the images that need to be loaded, and then load them. Try to find this request. Below is what it looks like on my end (click on preview to see the response)
<img width="1440" alt="Screenshot 2021-06-01 at 17 27 31" src="https://user-images.githubusercontent.com/10604097/120351073-d4f1d680-c2ff-11eb-9a03-cb1466602301.png">

6. What I'm interested in is the request itself, mainly the request payload. On my end this is what it looks like (this time click on headers)
<img width="1440" alt="Screenshot 2021-06-01 at 17 27 23" src="https://user-images.githubusercontent.com/10604097/120351188-f5219580-c2ff-11eb-9dd2-a6df18372de9.png">

Hopefully this shows us something.
